sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOIT.java

```diff
@@ -137,6 +140,11 @@ public void writeThenReadAll() {
             .apply(
                 "Gather read start time",
                 ParDo.of(new TimeMonitor<>(PARQUET_NAMESPACE, "readStart")))
+            .apply(
+                "Collect byte count", ParDo.of(new ByteMonitor<>(PARQUET_NAMESPACE, "readBytes")))
+            .apply(
+                "Collect element count",
+                ParDo.of(new CountMonitor<>(PARQUET_NAMESPACE, "elementCount")))
```

Comment:
We should name things like this concisely. In AvroIOIT this is an `itemCount`. Do you think we should have an enum for this stuff (names)?

Reply:
This also applies to other names like this

Reply:
I changed the name, thanks for the suggestion. 

How do you see an enum like this working? Would it have both internal metric names and column names for BQ?

Reply:
I added an enum, @lgajowy can you take a look?